std_msgs/Empty empty
---
float32 position1
float32 position2
float32 position3
float32 position4
float32 position5